otelcol-builder.yaml

```diff
@@ -0,0 +1,53 @@
+# https://github.com/open-telemetry/opentelemetry-collector-releases/blob/v0.120.0/distributions/otelcol/manifest.yaml
+
+dist:
+  name: our-super-cool-otelcol
+  description: Local OpenTelemetry Collector binary
+  output_path: ./_build
+
+receivers:
+  - gomod: go.opentelemetry.io/collector/receiver/nopreceiver v0.120.0
```

Comment:
Should the otelcol-builder file be duplicated across version folders?
This way version bumping could be trivial: we would have CI for pulling the matching `otelcol/manifest.yaml` and patch with our additions.